nix/modules/home/desktop/shells/noctalia/Readme.md

```
@@ -0,0 +1,22 @@
+# Niri/Noctalia
+
+## Theming
+
+Theme switching is handled by Noctalia. When the mode is switched, Noctalia:
+
+1. Updates its own settings.json file with colorSchemes.darkMode = true/false
+2. Updates any app's theme it is controlling.
+3. Runs userTemplates:
+  1. Updates a file that is used to determine dark/light (uses hex of bg color
```

markdownlint-cli2 (0.21.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)